packages/core/src/js/feedback/FeedbackFormScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import { captureFeedback } from '@sentry/core';
+import type { SendFeedbackParams } from '@sentry/types';
+import * as React from 'react';
+import type { KeyboardTypeOptions } from 'react-native';
+import { Alert, Text, TextInput, TouchableOpacity, View } from 'react-native';
+
+import defaultStyles from './FeedbackFormScreen.styles';
+import type { FeedbackFormScreenProps, FeedbackFormScreenState } from './FeedbackFormScreen.types';
+
+/**
+ * Implements a feedback form screen that sends feedback to Sentry using Sentry.captureFeedback.
+ */
+export class FeedbackFormScreen extends React.Component<FeedbackFormScreenProps, FeedbackFormScreenState> {
+  public constructor(props: FeedbackFormScreenProps) {
+    super(props);
+    this.state = {
```

Comment:
[I originally tried the `useState` hook for state](https://github.com/getsentry/sentry-react-native/pull/4328/commits/034efdea49b2da296842969a58e73c6819b788e7) but this resulted in an error due to multiple react instances.
```
 (NOBRIDGE) ERROR  Warning: Invalid hook call...
```
I decided to step back and implement a custom state to be more compliant with any client code.
Any feedback to simplify the code with hooks or other is more than welcome 🙇

Reply:
Thank you for the explanation.

I think the multiple React instances are issue of our monorepo setup.